### PR TITLE
doc: add issue template for creating a new issue

### PR DESCRIPTION
**Pull request by donutloop**
Hi there,

It's hard to solve a problem when important details are missing.

Ref: https://github.com/blog/2111-issue-and-pull-request-templates

Best regards,
Donutloop

### CODE REVIEW

**Comment by cryptocode** [comment]
Would it make sense to say something about *not* submitting security bugs here, and point such things to the bounty page?

**Comment by donutloop** [comment]
Actually, that is a good point. Shall I add paragraphs for that case?

**Comment by cryptocode** [comment]
@donutloop I personally think so, but the core devs should probably chime in on the wording.

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+BUG REPORT INFORMATION
+---------------------------------------------------
+-->
+
+**Description**
+
+<!--
+Briefly describe the problem you are having in a few paragraphs.
+-->
+
```
Comment (androm3da) [inline .github/ISSUE_TEMPLATE.md]: Please also include:

- indication whether this is a problem with the protocol, node or something else
- fields for which OS they are using
- description on how to get the logs for the raiblocks node
- warning not to disclose items for the Raiblocks bug bounty here, they must be communicated via the official channel!

**Comment by donutloop** [comment]
Is that ok?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+- (Linux) Kernel (e.g. `uname -a`):
+- Node version
+- (docker node) docker version 
+-->
+
+**logs**
+
+<!--
+
+Can you please provide the RaiBlocks logs for further analyze.
```
Comment (androm3da) [inline .github/ISSUE_TEMPLATE.md]: "further analyze" should be "further analysis"

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+<!--
+
+---------------------------------------------------
+BUG BOUNTY REPORT INFORMATION
+---------------------------------------------------
+
+WARNING! Don't disclose any information for the Raiblocks bug bounty. First, please read the following paragraphs.
+
+In the interest of further improving the security of the network, we have launched the RaiBlocks Bug Bounty Program.
```
Comment (androm3da) [inline .github/ISSUE_TEMPLATE.md]: Please don't reproduce the entire bug bounty info here.  Instead either refer them to the reddit or medium posts.

**Comment by donutloop** [comment]
that squash feature is cool!

**Comment by androm3da** [comment]
ok yes but there were other comments w/that review

**Comment by androm3da** [comment]
Please don't reproduce the entire bug bounty info here. Instead either refer them to the reddit or medium posts.



"further analyze" should be "further analysis"

**Comment by donutloop** [comment]
i changed that or what do you mean by that?

**Pull request merged**